vmdb/app/models/host.rb

```diff
@@ -1244,15 +1244,21 @@ def detect_discovered_os(ost)
   def detect_discovered_hypervisor(ost, ipaddr)
     log_header = 'MIQ(Host.detect_discovered_hypervisor)'
     find_method = :find_by_ipaddress
+
+    # TODO: break this apart, we probably want to only do the namelookup for hyperv, esx
+    hostname = Socket.getaddrinfo(ipaddr, nil)[0][2]
```

Comment:
Thanks... I didn't know about that.  I threw this up as a "OH MAN, we need to fix this" TODO item as I was looking for places still using ipaddress instead of hostname.  I'll update the description to have more information.